daemon/src/model/cfd.rs

```diff
@@ -1135,9 +1139,10 @@ impl Cfd {
 
         match evt.event {
             ContractSetupStarted => self.during_contract_setup = true,
-            ContractSetupCompleted { dlc } => {
+            ContractSetupCompleted { dlc, funding_fee } => {
                 self.dlc = Some(dlc);
                 self.during_contract_setup = false;
+                todo!("{funding_fee:?}");
```

Comment:
Unhandled TODO? This will blow up the program no?

Reply:
oops! of course this needs to be dealt with, thanks for spotting the error!